apollo-portal/src/main/java/com/ctrip/framework/apollo/openapi/v1/controller/ReleaseController.java

```diff
@@ -71,4 +80,72 @@ public OpenReleaseDTO loadLatestActiveRelease(@PathVariable String appId, @PathV
     return OpenApiBeanUtils.transformFromReleaseDTO(releaseDTO);
   }
 
+  @RequestMapping(value = "/apps/{appId}/clusters/{clusterName}/namespaces/{namespaceName}/branches/{branchName}/operator/{operator}/releases",
+          method = RequestMethod.POST)
+  public ReleaseDTO createGrayRelease(@PathVariable String appId,
+                                      @PathVariable String env, @PathVariable String clusterName,
+                                      @PathVariable String namespaceName, @PathVariable String branchName,
+                                      @PathVariable String operator,
```

Comment:
normally, operator should be a RequestParam(query parameter) instead of a PathVariable